tests/browser/test/test_setup.js

```diff
@@ -404,13 +404,11 @@ async function contextMenuSelect(browser, block, itemText) {
   // Clicking will always happen in the middle of the block's bounds
   // (including children) by default, which causes problems if it has holes
   // (e.g. statement inputs).
-  // Instead we want to click 10px from the left and 10px from the top.
-  const blockWidth = await block.getSize('width');
-  const blockHeight = await block.getSize('height');
-  const xOffset = -Math.round(blockWidth * 0.5) + 10;
-  const yOffset = -Math.round(blockHeight * 0.5) + 10;
+  // Instead, we'll click directly on the first bit of text on the block.
```

Comment:
I think you should do text as the first option and offset as a fallback, because not all blocks have text (especially field test blocks)

Reply:
We discussed offline. We don't need to be able to right-click on every type of block, and we already have the right-click functionality tested using blocks that do have tests, so this isn't that important.

It's reasonable to do the math as a fallback, but something is going weird in the second disable test, which causes the test to fail if you don't do the `waitForExist` (or a brief pause). We couldn't figure out why this would only happen in one of the tests and not all of them, or what was special about that one test. So in the interest of deflaking the current tests we're just going to submit as-is and worry about that little mystery at a future time. I did add a comment saying the block must have text on it.